### ISSUE TITLE
Support spotify control through MPD interface

### ISSUE THREAD

**Issue by lgnap** [volumio/Volumio2#1308]
Hello,

A lot of interface are available to control volumio (web, mpd, rpc, websockets) but in fact we don't know what is fully usable for all commands/sources.

I try to change volume through MPD client, that doesn't work until change the right option into config.
Next, previous doesn't work from mpd/mpc for spotify streaming.

I'm ready to implement a full MPD controller/interface to talk with the state machine if you need it.
I think that it will be very useful to base volumio on a lot of existing and proved clients (sometimes old, I should admit that)

**Comment by volumio** [comment]
Yes please, this interface is already there but it is not finished, if you could make it that would be just fantastic:
https://github.com/volumio/Volumio2/tree/master/app/plugins/user_interface/mpdemulation
